Tella/Scenes/CommonConnectionReport/ViewModels/DraftMainViewModel.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 import Combine
 
-class DraftMainViewModel<T: ServerProtocol>: ObservableObject {
+class DraftMainViewModel<T: Server>: ObservableObject {
```

Comment:
It's better to delete `<T: Server>` and declare the Server type directly 

`@Published var server :  Server?`